src/Tasks/Microsoft.NET.Build.Tasks/RunReadyToRunCompiler.cs

```diff
@@ -35,25 +35,34 @@ public class RunReadyToRunCompiler : ToolTask
 
         private bool IsPdbCompilation => !String.IsNullOrEmpty(_createPDBCommand);
 
+        private string DotNetHostPath => Crossgen2Tool?.GetMetadata("DotNetHostPath") ?? null;
```

Comment:
Does the `?? null` part do anything?

Reply:
That's obviously no-op, removed in 6th commit, thanks for spotting!